Bottom: c16054de0eb1e81356287fd39ba4cb6b5c839d23
Top:    3fe1a430b28d9deb750fd2bd4571ad4a55532d72
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-18 02:07:49 +0000

Refresh of find-c-add-flag-parameter-to-r

---

diff --git a/src/rats.c b/src/rats.c
index e1dca7e..17068a1 100644
--- a/src/rats.c
+++ b/src/rats.c
@@ -400,7 +400,7 @@ CheckShorts (LibraryMenuType *theNet)
   }
   ENDALL_LOOP;
   ALLPAD_LOOP (PCB->Data);
-  {CheckShorts
+  {
     if (TEST_FLAG (DRCFLAG, pad))
       {
 	warn = true;
